pymaker/auth.py

```diff
@@ -76,3 +76,37 @@ def permit(self, src, dst, sig: bytes) -> Transact:
 
     def __repr__(self):
         return f"DSGuard('{self.address}')"
+
+
+# TODO: Complete implementation and unit test
```

Comment:
IMO we should add in Readme that this is not in scope of this phase but any community contribution welcomed